### PR TITLE
WRN-17844: Add `locale` option for `forceDirection` prop of `MarqueeDecorator`

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
In the current Sandstone Marquee, if there is even one RTL character, the whole is treated as RTL. 
For bidirectional text, sometime we need to follow the direction of the locale.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
add `locale` prop for `forceDirection`prop of `ui/MarqueeDecorator`

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-17844

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

**Review by 0x64 — APPROVED** [review]

**Review by MikyungKim — COMMENTED** [review]
Please update your branch to the recent develop.

File: packages/ui/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The following is a curated list of changes in the Enact ui module, newest change
 
 - Updated to use `forwardCustom` and add `type` when forwarding custom events
 
+### Added
```
Comment (MikyungKim) [inline packages/ui/CHANGELOG.md:9]: Please put the log in the right place.

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Review by MikyungKim — COMMENTED** [review]
How about adding a unit test case for this?

File: packages/ui/Marquee/tests/Marquee-specs.js [hunk 2]
```diff
@@ -79,6 +79,19 @@ describe('Marquee', () => {
 		expect(marquee).toHaveStyle({'direction': expected});
 	});
 
+	test('should not switch directionality when forceDirection is "locale" and locale is LTR', () => {
```
Comment (MikyungKim) [inline packages/ui/Marquee/tests/Marquee-specs.js:82]: 😄 👍

**Review by MikyungKim — APPROVED** [review]
:shipit:

File: packages/ui/Marquee/tests/Marquee-specs.js (hunk 2 above)
Comment (MikyungKim) [inline packages/ui/Marquee/tests/Marquee-specs.js:82]:
```suggestion
	test('should not override direction to RTL when forceDirection is "locale" and locale is LTR', () => {
```

**Pull request merged**